src/components/BookList.js

```diff
@@ -1,9 +1,17 @@
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
 import Book from './Book';
+import { fetchBooks } from '../redux/books/booksSlice';
 // import books from './List';
 
 const BookList = () => {
   const books = useSelector((state) => state.books.value);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, [dispatch]);
+
   return (
     <div>
       {books.map((item) => (
```

Comment:
- [ ] Good job setting up the redux part of fetching boos as required, but the books are not shown instantly when added as expected. Every time a book is added, the page does not get updated instantly as a result of not using state values. Having a state to store fetched data helps re-render the page when the state's value changes. Currently, when a book is added or deleted, one must refresh the page to see the changes in the UI. Kindly work on that 👍🏽